### DISCUSSION TITLE
Plugin not listed in database

### DISCUSSION THREAD

**Discussion by ElijahElliotP** [gee-community/qgis-earthengine-plugin#94] [General]
When I choose 
    `Plugins --> Manage and Install Plugins...`
and then search for 'Google Earth' I do not find the plugin listed. I'm running QGIS 3.16.3.
I've tried going to
    `Settings --> Reload Repository`
but nothing changes. What might I be doing wrong?
![image](https://user-images.githubusercontent.com/28412116/114900347-c861ff00-9de1-11eb-8f07-9f21d64ee340.png)

**Comment by gena** [comment]
Strange, I can see it in 3.18.0. 

You can try to get it from the QGIS plugins page: https://plugins.qgis.org/plugins/ee_plugin/version/0.0.4/download/ but it should work via the plugins dialog as well, maybe something broke in the metainfo.

**Reply by ElijahElliotP** [reply]
So I used your link and installed directly from the zip and I see this: 
![image](https://user-images.githubusercontent.com/28412116/114907573-33fb9a80-9de9-11eb-91a9-4abf423d0460.png)

So I'll just go ahead and upgrade my installation, but I thought that was a bit odd.

**Reply by ElijahElliotP** [reply]
Found the issue. Entirely my fault. Thanks for your help!

**Reply by XavierCLL** [reply]
Hi @ElijahElliotP, could you describe what the problem was and how to fix it? it could help other people with the same issue in the future, thanks.

**Reply by ElijahElliotP** [reply]
Fair enough! 
Embarrassingly this is what can happen when you have an untidy system after multiple versions. For whatever reason I was clicking on an incorrect shortcut to launch (and I think seeing an incorrect splash screen, but now I don't trust my memory) for the version I was actually running, so my QGIS version actually was ineligible and everything was fine. I have no idea how my shortcuts and installations could get so badly confused but there you have it. 
Let this be a lesson to you all!
Cheers

p.s. maybe a mod could blow up this whole thread and spare me perpetual open-source embarrassment
